MainRepository/Program.cs

```diff
@@ -2,6 +2,26 @@
 {
     private static void Main(string[] args)
     {
-        Console.WriteLine("Hello, World!");
+        if (int.TryParse(args[0], out int n) && n > 0 && n <= 25) Console.WriteLine($"The {n}th armstrong number is {Armstrong(n)}");
+        else Console.WriteLine("Enter the valid input");

```

Comment:
Instead of saying enter the valid input, convey what the program expects. Eg: Max input is 25